src/js/common/schemaform/save-load-actions.js

```diff
@@ -161,14 +170,14 @@ export function fetchInProgressForm(formId, migrations) {
     // If we don't have a userToken, fail safely
     if (!userToken) {
       dispatch(setFetchFormStatus(LOAD_STATUSES.noAuth)); // Shouldn't get here, but just in case
-      return;
+      return Promise.reject('no auth'); // Returns rejected Promise for testing only
```

Comment:
I don't think that's necessary at the moment, since you should hit this if you aren't using save in progress already.